pkg/login/flow.go

```diff
@@ -167,29 +167,39 @@ func (f *Flow) triggerMethod(provider *Provider) (acsToken string, err error) {
 				if provider.ID == DCOSOIDCAuth0 {
 					loginServer, err = loginserver.New(startFlowURL)
 					if err != nil {
-						return "", err
+						f.logger.Error(err)
```

Comment:
Previously we returned at this point. Why this change?

Reply:
This lets use fallback to reading from stdin in that case.